packages/affiliates/libs/affiliates/devmining_v2.js

```diff
@@ -0,0 +1,250 @@
+const highland = require("highland");
```

Comment:
this is the devmining v2 business logic. Its almost completely identical to v1, but removes any calculations based on synthetic tokens, and just converts collateral to USD